tests/unit/commands/local/invoke/test_cli.py

```diff
@@ -73,6 +77,74 @@ def test_cli_must_setup_context_and_invoke(self, get_event_mock, InvokeContextMo
+                   debug_port=self.debug_port,
+                   debug_args=self.debug_args,
+                   debugger_path=self.debugger_path,
+                   docker_volume_basedir=self.docker_volume_basedir,
+                   docker_network=self.docker_network,
+                   log_file=self.log_file,
+                   skip_pull_image=self.skip_pull_image,
+                   profile=self.profile,
+                   region=self.region)
+
+        InvokeContextMock.assert_called_with(template_file=self.template,
+                                             function_identifier=self.function_id,
+                                             env_vars_file=self.env_vars,
+                                             docker_volume_basedir=self.docker_volume_basedir,
+                                             docker_network=self.docker_network,
+                                             log_file=self.log_file,
+                                             skip_pull_image=self.skip_pull_image,
+                                             aws_profile=self.profile,
+                                             debug_port=self.debug_port,
+                                             debug_args=self.debug_args,
+                                             debugger_path=self.debugger_path,
+                                             aws_region=self.region)
+
+        context_mock.local_lambda_runner.invoke.assert_called_with(context_mock.function_name,
+                                                                   event="{}",
+                                                                   stdout=context_mock.stdout,
+                                                                   stderr=context_mock.stderr)
+        get_event_mock.assert_not_called()
+
+    @patch("samcli.commands.local.invoke.cli.InvokeContext")
+    @patch("samcli.commands.local.invoke.cli._get_event")
+    def test_must_raise_user_exception_on_no_event_and_event(self, get_event_mock, InvokeContextMock):
+        self.no_event = True
+
+        with self.assertRaises(UserException) as ex_ctx:
+
+            invoke_cli(ctx=None,
+                       function_identifier=self.function_id,
+                       template=self.template,
+                       event=self.eventfile,
+                       no_event=self.no_event,
+                       env_vars=self.env_vars,
+                       debug_port=self.debug_port,
+                       debug_args=self.debug_args,
+                       debugger_path=self.debugger_path,
+                       docker_volume_basedir=self.docker_volume_basedir,
+                       docker_network=self.docker_network,
+                       log_file=self.log_file,
+                       skip_pull_image=self.skip_pull_image,
+                       profile=self.profile,
+                       region=self.region)
+
+        msg = str(ex_ctx.exception)
+        print(msg)
```

Comment:
Can we remove this print statement from the tests? Assuming you where using this during testing.

Reply:
Woops. My bad. I've fixed that.